app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -434,6 +419,14 @@ class NewMessageFragment : Fragment() {
         }
     }
 
+    private fun observeFromData() = with(newMessageViewModel) {
+        fromLiveData.observe(viewLifecycleOwner) { (signature, shouldUpdateUI) ->
+            updateSelectedSignatureInFromField(signature)
+            if (shouldUpdateUI) updateSelectedSignatureInSignatureField(signature)
+            signatureAdapter.updateSelectedSignature(signature.id)
```

Comment:
I find this bit unclear
Why shouldUpdateUi only concern the body signature ? The from field is also UI